src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ttpp/TimeToPayProxySteps.scala

```diff
@@ -40,6 +43,254 @@ class TimeToPayProxySteps extends ScalaDsl with EN with Eventually with Matchers
     TimeToPayProxyRequests.createPlanRequestBody(dataTable)
   }
 
+  Given("a create plan request with an empty customer reference") { () =>
```

Comment:
Is this already covered by 'Scenario TTP Promote an Arrangement request to create a plan for storage -empty customerReference'?

Reply:
Yeah - this step is to provide data for that scenario